plugin/src/App/init.lua

```diff
@@ -87,7 +90,65 @@ function App:getHostAndPort()
 	return host, port
 end
 
+function App:claimSyncLock()
+	if #Players:GetPlayers() == 0 then
+		Log.trace("Skipping sync lock because this isn't in Team Create")
+		return true
+	end
+
+	local lock = ServerStorage:FindFirstChild("Rojo_SessionLock")
+	if not lock then
+		lock = Instance.new("ObjectValue")
+		lock.Name = "Rojo_SessionLock"
+		lock.Archivable = false
+		lock.Value = Players.LocalPlayer
+		lock.Parent = ServerStorage
```

Comment:
ServerStorage does not feel like the right place to put this...

Reply:
@boatbomber, if people sync into serverstorage, it'll automatically be overwritten by rojo - defeating the point?

Reply:
We've been asking Roblox to give us PluginStorageService for years, but it still hasn't happened 🤷‍♂️